Tools/AP_Bootloader/board_types.txt

```diff
@@ -57,6 +58,7 @@ Reserved "PX4 [BL] FMU v6X.x"          53
 Reserved "PX4 [BL] FMU v6C.x"          56
 
 Reserved "ARK [BL] FMU v6X.x"          57
+Reserved "ARK [BL] FPV"                59
```

Comment:
this causes CI to fail

Reply:
yeah seems like a pretty low id

Reply:
The ID is already set in PX4 and QGC. Why would the board ID make the CI fail?

Reply:
IDK, one for @peterbarker

Reply:
looks like bl/fmu don't share IDs
https://github.com/ArduPilot/ardupilot/blob/master/Tools/autotest/validate_board_list.py#L55-L66